pytorch_lightning/loggers/neptune.py

```diff
@@ -296,113 +565,46 @@ def version(self) -> str:
 
     @rank_zero_only
-    def log_text(self, log_name: str, text: str, step: Optional[int] = None) -> None:
-        """
-        Log text data in Neptune experiments.
-
-        Args:
-            log_name: The name of log, i.e. mse, my_text_data, timing_info.
-            text: The value of the log (data-point).
-            step: Step number at which the metrics should be recorded, must be strictly increasing
-        """
-        if step is None:
-            self.experiment.log_text(log_name, text)
-        else:
-            self.experiment.log_text(log_name, x=step, y=text)
+    def log_text(self, *args, **kwargs):
+        self._raise_deprecated_api_usage("log_text", f"logger.run['{self._prefix}/key'].log('text')")
 
     @rank_zero_only
-    def log_image(self, log_name: str, image: Union[str, Any], step: Optional[int] = None) -> None:
-        """
-        Log image data in Neptune experiment
-
-        Args:
-            log_name: The name of log, i.e. bboxes, visualisations, sample_images.
-            image: The value of the log (data-point).
-                Can be one of the following types: PIL image, `matplotlib.figure.Figure`,
-                path to image file (str)
-            step: Step number at which the metrics should be recorded, must be strictly increasing
-        """
-        if step is None:
-            self.experiment.log_image(log_name, image)
-        else:
-            self.experiment.log_image(log_name, x=step, y=image)
+    def log_image(self, *args, **kwargs):
+        self._raise_deprecated_api_usage("log_image",
+                                         f"logger.run['{self._prefix}/key'].log(File('path_to_image'))")
 
     @rank_zero_only
-    def log_artifact(self, artifact: str, destination: Optional[str] = None) -> None:
-        """Save an artifact (file) in Neptune experiment storage.
-
-        Args:
-            artifact: A path to the file in local filesystem.
-            destination: Optional. Default is ``None``. A destination path.
-                If ``None`` is passed, an artifact file name will be used.
-        """
-        self.experiment.log_artifact(artifact, destination)
+    def log_artifact(self, *args, **kwargs):
+        self._raise_deprecated_api_usage("log_artifact",
+                                         f"logger.run['{self._prefix}/{self.ARTIFACTS_KEY}/key'].log('path_to_file')")
 
     @rank_zero_only
-    def set_property(self, key: str, value: Any) -> None:
-        """
-        Set key-value pair as Neptune experiment property.
-
-        Args:
-            key: Property key.
-            value: New value of a property.
-        """
-        self.experiment.set_property(key, value)
+    def set_property(self, *args, **kwargs):
+        self._raise_deprecated_api_usage("log_artifact",
+                                         f"logger.run['{self._prefix}/{self.PARAMETERS_KEY}/key'].log(value)")
```

Comment:
formatting broken here?

Reply:
Why?

Reply:
I mean it's not according to our formatting rules, there is arbitrary empty space here. 
Can you run black on the file? See also the pre-commit hook is failing :)

Reply:
Yes, I've just fixed all issues locally and commited changes. Thanks for info.

Reply:
ah, you are first time contributor so you didn't see our failed checks sorry. 
we had to start the CI manually (always when new contributor sends a PR).